core/conn/trafci/src/main/java/org/trafodion/ci/ConsoleReader.java

```diff
@@ -172,7 +176,8 @@ public String readLine() throws UnsupportedEncodingException, UserInterruption,
 
```

Comment:
This will remove Ctrl-C signal handler, did you test Ctrl-C in trafci? It should do a 'cancel query' if I remember it correctly. Anyway, you should make sure Ctrl-C has same behavior as before, should not be broken by this change. 
I am not sure, need to check the trafci manual, but wondering if you have tested it or not? Since you comments out the signal handler.